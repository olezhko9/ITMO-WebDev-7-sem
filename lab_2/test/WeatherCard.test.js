import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

import { shallow, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import WeatherCard from '../src/components/WeatherCard';



describe('Weather Card', () => {

  it('renders correctly when data is loading', () => {
    const cityWeather = {
      name: 'Moscow',
      isLoading: true
    }

    const tree = render(
      <WeatherCard
        cityWeatherData={cityWeather}
        isFavorite
      />);
    expect(shallowToJson(tree)).toMatchSnapshot();
  });



})
