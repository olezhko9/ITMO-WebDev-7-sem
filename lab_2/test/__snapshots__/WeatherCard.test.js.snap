// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather Card renders correctly when data is loading 1`] = `null`;
